sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -4,9 +4,7 @@
   "path": "sfaa2021-22",
   "type": "form",
   "display": "wizard",
-  "tags": [
-    "common"
-  ],
+  "tags": ["common"],
```

Comment:
Just to confirm, are you using the JSON formatter.?

Reply:
I'm using the plugins from the development standards page, including Prettier, though I'm not sure _which_ plugin made that change. It wasn't done manually.